kroto-plus-gradle-plugin/src/main/kotlin/com/github/marcoferrer/krotoplus/gradle/protobuf/GenericProtobufConfigurator.kt

```diff
@@ -0,0 +1,114 @@
+/*
```

Comment:
This file introduces preliminary work needed for simplifying protobuf configuration for generic use cases